regions/core/core.py

```diff
@@ -98,11 +89,9 @@ def __repr__(self):
 
     def __str__(self):
         cls_info = [('Region', self.__class__.__name__)]
-        if hasattr(self, 'center'):
-            cls_info.append(('center', self.center))
-        if self._repr_params is not None:
+        if self._fields is not None:
```

Comment:
I have a slight preference for `_params` over `_fields`, but no big deal.

Reply:
Changed to `_params`.